hack/make_local.sh

```diff
@@ -21,7 +21,7 @@ sed -i "s|-f https://raw.githubusercontent.com/kubevirt/hyperconverged-cluster-o
 
 chmod +x _local/deploy.sh
 
-ln -s assets/upgradePatches.json upgradePatches.json
+ln -sf assets/upgradePatches.json upgradePatches.json
```

Comment:
Without the force flag, `make local` would fail with below error if you try it second time.
```sh
+ ln -s assets/upgradePatches.json upgradePatches.json
ln: failed to create symbolic link 'upgradePatches.json': File exists
make: *** [Makefile:214: local] Error 1
```